.github/workflows/dashboard-api-tests.yml

```diff
@@ -0,0 +1,17 @@
+name: Dashboard Api Tests
+on:
+  push:
+    paths:
+      - "dashboard-api/**"
+
+jobs:
+  tests:
+    name: Tests
+    runs-on: ubuntu-20.04
+    env:
+      working-directory: ./dashboard-api
+    steps:
+      - uses: actions/checkout@v2
+      - name: Tests
+        run: ./test.sh --is-ci
+        working-directory: ${{env.working-directory}}
```

Comment:
Are we using `working-directory` env var somewhere else besides from line 17? If not, couldn't we just use `./dashboard-api` instead of `${{env.working-directory}}`?

Reply:
I used the other github action I have as a baseline and I didn't remind to change this. Good catch 👍